iroha-cli/interactive/impl/interactive_common_cli.cpp

```diff
@@ -87,12 +97,20 @@ namespace iroha_cli {
       return line;
     }
 
+    boost::optional<std::string> promptString(ParamData param) {
+        std::string message = std::get<0>(param);
```

Comment:
I consider adding a new structure, because `std::get` say nothing about the semantic